controllers/datadogagent/feature/utils/utils.go

```diff
@@ -18,7 +18,7 @@ import (
 
 // Process Checks utils
 
-const RunInCoreAgentMinVersion = "7.53.0-0"
+const RunInCoreAgentMinVersion = "7.57.0-0"
```

Comment:
I just updated this version to the latest that supports language detection in the core agent and I think we should continue to update this until we GA (this is not a publicly documented feature yet). The code would be simplified as a result and I think there would be less headache in the future as we would need to worry about less versions. Curious if there any reasons not to do this.

Reply:
Adding for further context: this version check impacts the following: Process Collection, Process Discovery, Container Collection, and now Language Detection/APM SSI. The config itself controls whether these features run on the process agent or the core agent.

Reply:
> The only concern I can think of is negatively impacting our internal deployments using this config, but 7.57.0 should already be rolled out by then.

Will this impact beta customers for the run-in-core-agent feature which will be on 7.56?

Reply:
I dont think so. They would be on Operator 1.8.

Reply:
And, once operator 1.9 is released, I imagine 7.57 should already be released.

Reply:
What happens to the customers who have Agent version pinned to pre-7.57 version and upgrade to the new version of Operator?

Reply:
If the `RunInCoreAgent` feature is enabled then their agent deployment would revert to the prior setup. I think thats ok because this is not a documented feature for the Operator yet and where it is documented, its descirbed as experimental. Otherwise, nothing should happen.